catalog/ec2-policies.yaml

```diff
@@ -1,6 +1,7 @@
 #
 # This policy denies instance types that aren't based on the Nitro system as documented in the following document:
 # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-types.html#ec2-nitro-instances
+# List of Nitro-enabled instances updated 2023-06-23
```

Comment:
```suggestion
# List of Nitro-enabled instances updated 2023-06-23
#
# aws ec2 describe-instance-types \
# --filters Name=hypervisor,Values=nitro \
# --query "InstanceTypes[*].[InstanceType]" --output text | sort
```

Reply:
The following should match the list returned by this command

Reply:
Done. Though see comment below about missing metal instances.